tests/test_icgc.py

```diff
@@ -1,5 +1,6 @@
-from lib.cli import icgc_download_client
 from argparse import Namespace
+
+from icgcget import cli
```

Comment:
Please add source headers to all files and add this to your IDE templates. Thanks.